src/repository.ts

```diff
@@ -479,11 +483,13 @@ export class Repository {
           /(.+?)\.(mine|working|merge-\w+\.r\d+|r\d+)$/
         );
 
-        // If file end with (mine, working, merge, etc..) and has file without extension
+        // If file end with (mine, working, merge, etc..), has file without extension and
+        // sourceControl.hideUnversioned flag is turned on.
         if (
           matches &&
           matches[1] &&
-          statuses.some(s => s.path === matches[1])
+          statuses.some(s => s.path === matches[1]) &&
+          hideUnversioned
```

Comment:
This doesn't hide the unversioned changes for me

Reply:
could just do 
`if (hideUnversioned) {
          continue;
        }`

before the matches if?